src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discount3To6Days = 20;
+  const dailyRentalCost = 40;
+  const discount7OrMoreDays = 50;
+
+  if (days >= 3 && days < 7) {
+    return days * dailyRentalCost - discount3To6Days;
+  }
+
+  if (days >= 7) {
+    return days * dailyRentalCost - discount7OrMoreDays;
```

Comment:
As days * dailyRentalCost is always a constant- I would create a separate variable for that